File: api/health.js
export default async function handler(req, res) {
  res.setHeader('Access-Control-Allow-Origin', '*');
  res.setHeader('Access-Control-Allow-Headers', 'Content-Type');
  res.setHeader('Access-Control-Allow-Methods', 'GET, POST, OPTIONS');
  if (req.method === 'OPTIONS') return res.status(204).end();
  return res.status(200).json({ ok: true, has_api_key: Boolean(process.env.OPENAI_API_KEY) });
}

File: api/health.py
import json
import os

def handler(request):
    return {
        "statusCode": 200,
        "headers": {
            "content-type": "application/json; charset=utf-8",
            "access-control-allow-origin": "*"
        },
        "body": json.dumps({"ok": True, "has_api_key": bool(os.getenv("OPENAI_API_KEY"))})
    }
